frontend/src/components/feed/Feed.cy.js

```diff
@@ -15,13 +15,38 @@ describe("Feed", () => {
         })
       }
     ).as("getPosts")
-
     cy.mount(<Feed navigate={navigate}/>)
-    
     cy.wait("@getPosts").then(() =>{
       cy.get('[data-cy="post"]')
       .should('contain.text', "Hello, world")
       .and('contain.text', "Hello again, world")
     })
   })
+
+it("it displays a list of all the post links", () => {
```

Comment:
not sure if this is the review display, but the indentation looks very weird here. Are these tests nested in the describe("Feed", () => {} above?